clients/java/src/main/java/io/camunda/client/api/command/CompleteJobCommandStep1.java

```diff
@@ -89,5 +90,74 @@ interface CompleteJobCommandStep2 extends FinalCommandStep<CompleteJobResponse>
      * @return the builder for this command.
      */
     CompleteJobCommandStep2 denied(boolean denied);
+
+    /**
+     * Attributes that were corrected by the worker. The below attributes can be corrected and
+     * additional attributes will be ignored: `assignee` - reset by providing an empty String
+     * `dueDate` - reset by providing an empty String `followUpDate` - reset by providing an empty
+     * String `candidateGroups` - reset by providing an empty list `candidateUsers` - reset by
+     * providing an empty list `priority` - minimum 0, maximum 100, default 50 Omitting any of the
+     * attributes will preserve the persisted attribute's value.
```

Comment:
🔧 It looks like the auto-formatting affected the JavaDoc formatting here. To preserve readability, you can use HTML list tags or line breaks with `{@code}`:

1. Using HTML list tags:

   ```
   * <ul>
   *   <li><code>assignee</code> - reset by providing an empty String</li>
   *   <li><code>dueDate</code> - reset by providing an empty String</li>
   *   <li><code>followUpDate</code> - reset by providing an empty String</li>
   *   <li><code>candidateGroups</code> - reset by providing an empty list</li>
   *   <li><code>candidateUsers</code> - reset by providing an empty list</li>
   *   <li><code>priority</code> - minimum 0, maximum 100, default 50</li>
   * </ul>
   ```

2. **Using line breaks with `{@code}`:**

   ```java
   * {@code assignee} - reset by providing an empty String<br>
   * {@code dueDate} - reset by providing an empty String<br>
   ```

3. **Using `@link` for strong references (if appropriate):**

   ```java
   * {@link CompleteJobCommandStep3#assignee} - reset by providing an empty String<br>
   * {@link CompleteJobCommandStep3#dueDate} - reset by providing an empty String<br>
   ```

Each option has its pros and cons - HTML lists improve readability, `{@code}` is concise, and `@link` adds strong references but may be verbose.

Feel free to experiment and choose the one that best suits your needs. 😊